crates/agglayer-storage/src/stores/state/mod.rs

```diff
@@ -514,10 +511,19 @@ impl StateReader for StateStore {
     /// small. This function is only called once when the node starts.
     /// Benchmark: `last_certificate_bench.rs`
     fn get_active_networks(&self) -> Result<Vec<NetworkId>, Error> {
+        let disabled_networks = self
+            .db
+            .keys::<crate::columns::disabled_networks::DisabledNetworksColumn>()?
+            .filter_map(|v| v.ok())
```

Comment:
Should we error early here if there is some issue with reading disabled networks column? We don't want to consider it active if it is disabled by purpose.